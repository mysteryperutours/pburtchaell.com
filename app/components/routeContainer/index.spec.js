import React from 'react';
import RouteContainer from './index';

describe(RouteContainer.name, () => {
  const DEFAULT_CHILD = 'foo';

  it('should render component', () => {
    expect(shallow(
      <RouteContainer>
        {DEFAULT_CHILD}
      </RouteContainer>
    )).toMatchSnapshot();
  });
});
