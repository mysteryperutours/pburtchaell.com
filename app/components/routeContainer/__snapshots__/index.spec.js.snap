exports[`RouteContainer should render component 1`] = `
<div
  className="route-container">
  <RouteHeader
    align="left"
    initialBlueChannel={255}
    initialGreenChannel={255}
    initialRedChannel={255}
    isInterpolated={false} />
  <main
    className="undefined"
    role="main">
    foo
  </main>
  <RouteFooter
    color={
      Object {
        "b": 255,
        "g": 255,
        "r": 255,
      }
    } />
</div>
`;
